buildpack.md

```diff
@@ -1035,6 +1033,7 @@ homepage = "<buildpack homepage>"
 clear-env = false
 description = "<buildpack description>"
 keywords = [ "<string>" ]
+sbom = [ "<string>" ]
```

Comment:
Minor: At first glance the key was a little too generic that I thought I could somehow add additional sbom data via this key when in reality it's more about defining supported media types.

I would propose changing it to something like `sbom-types`.

Reply:
@samj1912 WDYT?

Reply:
`sbom-formats` maybe?

Reply:
Please see 12aff6f